core/math/basis.cpp

```diff
@@ -557,28 +557,9 @@ void Basis::set_euler_yxz(const Vector3 &p_euler) {
 	*this = ymat * xmat * zmat;
 }
 
-bool Basis::is_equal_approx(const Basis &a, const Basis &b, real_t p_epsilon) const {
```

Comment:
The epsilon is needed afaik because it can be used with different values for different use cases, when matrix is not of unit size. Check those that use is_equal_approx_ratio and leave them alone.

Reply:
Should I leave a note that it would be worth renaming this method to `is_equal_approx_ratio` in 4.0? It would be strange if Basis had different behavior for `is_equal_approx`.